modules/weblog/README.md

```diff
@@ -261,6 +261,29 @@ we still can get some info from them.
             match: '* *'
 ```
 
+## Custom Time Fields Feature
+
+Weblog is also able to extract user defined time fields and could count min/avg/max + histogram against these fields.
+
+This feature needs:
+-   custom log format with user defined time fields
+-   histogram to show response time in seconds which is optional.
+
+There is an example in apache custom log forma with 1 custom time fields - `^FB` which shows delay in microseconds between when the request arrived and the first byte of the response headers are written. Like Custom Fields feature Weblog is unaware of these fields, but
```

Comment:
forma -> format

Please split this into several lines.

@joelhans please take a look.

Reply:
I meant that you shouldn't use very long lines. You should try to keep every line less than 120 characters.